### DISCUSSION TITLE
How to decrease image caption size?

### DISCUSSION THREAD

**Discussion by Claus1** [quasarframework/quasar#11248] [General - Components / Directives / etc] [answered]
Standart image capture is huge and it is not clear how to make it 2x smaller in height.
![image](https://user-images.githubusercontent.com/1247062/140689225-d755daf0-b49d-4f27-b39a-7a08343ea510.png)
Can you point me right direction for it?

**Comment by metalsadman** [comment, marked as answer]
use css, either inline style or a class.
https://codepen.io/metalsadman/pen/LYjmNdJ
